app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      include Wor::Paginate
+
+      before_action :authenticate_user!
+      before_action :set_api_v2_book, only: [:show]
+
+      # GET /api/v2/books
+      def index
+        @api_v2_books = Book.all
+
+        render_paginated @api_v2_books
+      end
```

Comment:
No usamos variables de instancia en los controladores, solo en las memoizaciones. Este método podría quedar simplemente así

```suggestion
      def index
        render_paginated Book.all
      end
```

O así

```suggestion
      def index
        books = Book.all

        render_paginated books
      end
```

Reply:
done!